proxy/proxy_test.go

```diff
@@ -135,7 +136,12 @@ func newTestProxyWithFiltersAndParams(fr filters.Registry, doc string, params Pa
 		FilterRegistry: fr,
 		PollTimeout:    sourcePollTimeout,
 		DataClients:    []routing.DataClient{dc},
-		Log:            tl})
```

Comment:
Are you sure you want to add these predicates and these are not from another pr?

Reply:
yes, they are used in the test `TestTracingProxySpanWithRetry`. Do you think they can cause an issue in other tests?

Reply:
actually, i think it should be fine, it's just additional predicates, so yes :smile:

Reply:
I just ask, because I wondered that you modified a test and I always ask by test modifications that are not obvious.